README.md

```diff
@@ -16,6 +16,9 @@ The action packages, signs, and uploads the application to the specified ECR and
 | ecr-repo-name              | true     | The secret with the name of the ECR repo created by the app-container-repository stack | app-container-repository-tobytraining-containerrepository-i6gdfkdnwrrm               |
 | dockerfile                 | false     | The Dockerfile to use for the build | Dockerfile
 | checkout-repo                 | false     | Checks out the repo as the first step of the action. Default "true". | "true"
+| private-docker-registry | false | Private Docker registry URL. Default to "" | "abc12345.live.dynatrace.com"
```

Comment:
where is it getting the dynatrace url ?

Reply:
In absence of a good private docker registry example url, I have fashioned a random url from my current use case.